internal/pkg/archer/env.go

```diff
@@ -4,9 +4,10 @@
 // Package archer contains the structs that represent archer concepts, and the associated interfaces to manipulate them.
 package archer
 
-// Environment represents the configuration of a particular Environment in a Project. It includes
-// the location of the Environment (account and region), the name of the environment, as well as the project
-// the environment belongs to.
+import "github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy"
```

Comment:
I'm a tiny bit nervous adding dependencies to this pkg - because now we can't use any of these domains in the deploy pkg. But since it's mostly an interface pkg, that should be ok!

Reply:
Yeah, I realized that I view the EnvironmentDeployer interface as temporary. It should most likely move within the CLI package since it's the only place where it's used :P